### PR TITLE
feat(live): add apollo-link example

### PR DESCRIPTION
**Pull request by vitalikda**
# Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.

# Type of change

- [ ] 💔 Breaking
- [x] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

**Comment by notrab** [comment]
I notice here we're using pnpm and other examples Yarn.

Do we want to default to pnpm for external users too? Or stick with NPM to less to setup/configure. I feel like if someone knows how to run pnpm, they will instead of NPM.

File: examples/react-apollo-live/.env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Refer to the README on how to fill these variables
+VITE_GRAFBASE_API_URL=
+VITE_CLERK_FRONTEND_API=
```
Comment (notrab) [inline examples/react-apollo-live/.env.example:3]: We should maybe include a comment at the top of this file that the Clerk URL must be the full absolute URL, so we can remove the protocol from the `schema.graphql` and only rely on the true ENV value threre.

File: examples/react-apollo-live/grafbase/schema.graphql [hunk 2]
```diff
@@ -0,0 +1,13 @@
+schema
+  @auth(
+    providers: [{ type: oidc, issuer: "https://{{ env.CLERK_ISSUER_URL }}" }]
```
Comment (notrab) [inline examples/react-apollo-live/grafbase/schema.graphql:3]:
```suggestion
    providers: [{ type: oidc, issuer: "{{ env.CLERK_ISSUER_URL }}" }]
```

We don't tend to do this in other examples or documentation. Instead we rely on using environment variables for absolute values.

**Review by notrab — CHANGES_REQUESTED** [review]
Hey @vitalikda 

This is awesome. Nice job!

I've left a few comments which we should also apply to the URQL example too.

Having to login to view the guestbook is quite annoying but I know we've a product limitation that prevents up from making it public. So here's what I suggest we could potentially do to make it a smoother experience for new users:

1. Create a default JWT that expires way in the future. With a group that has access to "Read and Write".
2. Create another JWT with a group with operation permissions to "Read only"
3. Add some buttons to the top of the page that switches the current active token &mdash; this showcases group-based auth too!
4. Done

An example of the buttons to switch "current user" token can be seen in this video: https://youtu.be/vwUGlwbdfvI?t=47

File: examples/react-apollo-live/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Grafbook: Server sent events (SSE) x Clerk ⨯ React
+
+## Getting Started
+
+1. Run `npx degit grafbase/grafbase/examples/react-clerk-live grafbase-with-react-clerk-live` to clone this example
+1. Change directory into the new folder `cd grafbase-with-react-clerk-live`
+1. Run `cp .env.example .env` to copy the example `.env.example` file to `.env` in the root of your project
+1. Open `.env` in your code editor, and provide your Grafbase API endpoint and [Clerk frontend and backend API keys](https://dashboard.clerk.dev/last-active?path=api-keys)
```
Comment (notrab) [inline examples/react-apollo-live/README.md:8]: We could remove this step entirely for demo purposes if we used the JWT provider. Let me know what you think about the suggestion below in the review comments.

File: examples/react-apollo-live/.env.example (hunk 1 above)
Comment (vitalikda) [reply examples/react-apollo-live/.env.example:3]: Should we? I find it easier to copy-paste exactly how Clerk provides URL in their dashboard

Also, note that `VITE_CLERK_FRONTEND_API` is used in ClerkProvider without URL protocol, so we would have two versions of the same thing 
- `VITE_CLERK_FRONTEND_API` without protocol 
- `CLERK_ISSUER_URL` with protocol

Comment (notrab) [reply examples/react-apollo-live/.env.example:3]: I think we should yeah. They are separate `.env` files after all, so you'd have the same duplication anyways.

File: examples/react-apollo-live/README.md (hunk 3 above)
Comment (vitalikda) [reply examples/react-apollo-live/README.md]: done

File: examples/react-apollo-live/package.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "apollo-link-ts",
+  "private": true,
+  "version": "0.0.0",
```
Comment (notrab) [inline examples/react-apollo-live/package.json:4]: maybe remove `name` and `version`?

File: examples/react-apollo-live/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Grafbook: Server sent events (SSE) x JWT ⨯ React
```
Comment (vitalikda) [inline examples/react-apollo-live/README.md:1]:
```suggestion
# Grafbook: Server sent events (SSE) x Apollo ⨯ React
```

File: examples/react-apollo-live/package.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "apollo-link-ts",
```
Comment (vitalikda) [inline examples/react-apollo-live/package.json:2]:
```suggestion
  "name": "apollo-live-example",
```

File: examples/react-apollo-live/package.json (hunk 4 above)
Comment (vitalikda) [reply examples/react-apollo-live/package.json:4]: updated name

**Pull request merged**